packages/synapse-react-client/src/components/Markdown/MarkdownSynapse.tsx

```diff
@@ -419,9 +419,19 @@ const MarkdownSynapse: MarkdownSynapseComponent = class MarkdownSynapse extends
           value = attribute.value
         }
         if (name && value) {
-          props[name] = value
+          rawProps[name] = value
         }
       }
+
+      const { style: styleString, class: className, ...props } = rawProps
+      // Remap class to className
+      props.className = className
+      if (styleString) {
+        // React expects the `style` prop to be an object, not a string, so use
+        // the all-caps STYLE to pass the style string as a custom attribute
+        props.STYLE = styleString
+      }
```

Comment:
Fix issue where style wouldn't get passed along (we could parse it, but just passing the string along was easier)